.github/workflows/check-links.yaml

```diff
@@ -1,4 +1,4 @@
-name: check-links
+name: "Link Check"
```

Comment:
FYI, also changed the name of the workflow because this one is the only workflow not following this format.